test/cassava/client_test.rb

```diff
@@ -47,9 +47,31 @@ def string_keys(hash)
         item = { :id => 'i', :a => 1, :b => 'b', :c => "'\"item(", :d => 1, :ttl => ttl }
         @client.insert(:test, item)
 
-        assert @client.send(:insert_statement, :test, item, ttl).cql =~ /\sUSING\sTTL\s#{ttl}$/
+        assert @client.send(:insert_statement, :test, item, ttl).cql =~ /\sUSING\sTTL\s#{ttl}/
         assert_equal string_keys(item), @client.select(:test).execute.first
       end
+
+      should 'allow the insertion with a timestamp' do
+        timestamp = Time.now.to_i
```

Comment:
More of a concern for core, but looking at the cassandra docs, timestamp should be milliseconds. `Time.now.to_i` will give you seconds, so we want to make sure we're calling this correctly.